automation/tests-baremetal-utils.sh

```diff
@@ -0,0 +1,51 @@
+VETH_ETH1_CREATED="false"
+VETH_ETH2_CREATED="false"
+
+function check_iface_exist {
+    ip a s | grep -q $1
+}
+
+function prepare_baremetal_network {
```

Comment:
`prepare_baremetal_network` should be use the same command with container did. Some thing like

```
fun setup_test_nics {
 if [ $NMSTATE_TEST_NIC1 ] and [$NMSTATE_TEST_NIC2]; then
   ip link rename $NMSTATE_TEST_NIC1 to eth1
   ip link rename $NMSTATE_TEST_NIC2 to eth2
else:
  create veth eth1 and eth2 and set them as managed.
 fi
}

exec_cmd setup_test_nics

Reply:
Done, they are using the same function now.